src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Packaging/ManagedProjectSystemPackage.cs

```diff
@@ -35,13 +35,15 @@ internal partial class ManagedProjectSystemPackage : AsyncPackage
         public const int DebugFrameworksCmdId = 0x3050;
 
         public const string ManagedProjectSystemOrderCommandSet = "{6C4806E9-034E-4B64-99DE-29A6F837B993}";
-        public const int MoveUpCmdId = 0x2000;
-        public const int MoveDownCmdId = 0x2001;
         public const int AddNewItemAboveCmdId = 0x2002;
         public const int AddExistingItemAboveCmdId = 0x2003;
         public const int AddNewItemBelowCmdId = 0x2004;
         public const int AddExistingItemBelowCmdId = 0x2005;
 
+        public const string FSharpProjectCmdSet = "{75AC5611-A912-4195-8A65-457AE17416FB}";
```

Comment:
I'd suggest a comment here saying these are duplicated from the old project system, so that people don't change them.

Reply:
done